app/src/main/java/ch/epfl/sdp/blindly/MapsActivity.kt

```diff
@@ -0,0 +1,56 @@
+package ch.epfl.sdp.blindly
+
+import android.location.Location
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import ch.epfl.sdp.blindly.location.AndroidLocationService
+import ch.epfl.sdp.blindly.location.LocationService
+
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
+
+    private lateinit var mMap: GoogleMap
+    private var location: Location? = null
+    private lateinit var locSer: LocationService
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        locSer = AndroidLocationService(this)
+        location = locSer.getCurrentLocation()
+
+        setContentView(R.layout.activity_maps)
+        // Obtain the SupportMapFragment and get notified when the map is ready to be used.
+        val mapFragment = supportFragmentManager
+            .findFragmentById(R.id.map) as SupportMapFragment
+        mapFragment.getMapAsync(this)
+    }
+
+    /**
+     * Manipulates the map once available.
+     * This callback is triggered when the map is ready to be used.
+     * This is where we can add markers or lines, add listeners or move the camera. In this case,
+     * we just add a marker near Sydney, Australia.
+     * If Google Play services is not installed on the device, the user will be prompted to install
+     * it inside the SupportMapFragment. This method will only be triggered once the user has
+     * installed Google Play services and returned to the app.
+     */
+    override fun onMapReady(googleMap: GoogleMap) {
+        mMap = googleMap
+
+        val loc: LatLng = if(location != null) {
+            LatLng(location!!.latitude, location!!.longitude)
+        } else {
+            LatLng(46.5, 6.6)
```

Comment:
What is LatLng(46.5, 6.6) referring to ? The marker near Sydney ? Might want to add a comment or a constant

Reply:
The comment above explains this but it is not Sydney but Lausanne instead